GFX Button Example for Adafruit TFT FeatherWing - 3.5"

ChuckVanzant opened the pull request:
This example file creates a primitive menuing example using Adafruit_GFX_Button buttons in landscape mode and was developed using the Adafruit TFT FeatherWing - 3.5" 480x320 Touchscreen for Feathers. 

This example provides a two-level menuing system using up/select/down/return buttons. The first level controls the text size, where the text can be increased or decreased. The text color can be changed at the second menu level.

It has been tested on the Adafruit HUZZAH32 – ESP32 Feather Board and the Adafruit Feather HUZZAH with ESP8266. Please note that the display operations are noticeably slower with the ES8266 Feather.

ChuckVanzant committed:
GFX Button Example for Featherwing 3.5"

This example file creates a primitive menuing example using Adafruit_GFX_Button buttons in landscape mode and was developed using the Adafruit TFT FeatherWing - 3.5" 480x320 Touchscreen for Feathers. 

This example provides a two-level menuing system using up/select/down/return buttons. The first level controls the text size, where the text can be increased or decreased. The text color can be changed at the second menu level.

It has been tested on the Adafruit HUZZAH32 – ESP32 Feather Board and the Adafruit Feather HUZZAH with ESP8266. Please note that the display operations are noticeably slower with the ES8266 Feather.

ladyada commented:
hiya please add the other #define's for 32u4 and zero feather so travis passes. for due you can put a skip file (see the other featherwing examples) thx :)

ChuckVanzant committed:
Fix compile errors after edits.

Some replace all edits corrupted the compilation. The corrupted text was restored.

ChuckVanzant committed:
Update gfxbuttontest_featherwing.ino

ChuckVanzant committed:
Minor format edits

Only minor format edits were made.

ChuckVanzant committed:
Added gfxbuttontest_featherwing example

ChuckVanzant committed:
Added gfxbuttontest_featherwing example

Created gfxbuttontest_featherwing example in a gfxbuttontest_featherwing folder.

ChuckVanzant committed:
Delete .ino file in example folder

The .ino file in example folder is being deleted and recreated in examples/gfxbuttontest_featherwing/

ChuckVanzant committed:
Create   .mega2560.test.skip

ChuckVanzant committed:
Delete .mega2560.test.skip

ChuckVanzant committed:
Create   .mega2560.test.skip

ChuckVanzant committed:
Add additional board defines

Updated file by adding additional board defines.

ChuckVanzant committed:
Update .mega2560.test.skip

ChuckVanzant committed:
Update .mega2560.test.skip

makermelissa reviewed (approved):
Very cool. Tested and it works great. Thanks!

Nikodrengen commented:
Thank you Melissa, The example is clear and works well.
